src/CR.WorkingDayService/WorkingDayServiceBuilder.cs

```diff
@@ -0,0 +1,96 @@
+﻿// <copyright file="WorkingDayServiceBuilder.cs" company="Cognisant">
+// Copyright (c) Cognisant. All rights reserved.
+// </copyright>
+
+namespace CR.WorkingDayService
+{
+    using System.Collections.Generic;
+    using System.Linq;
+
+    /// <summary>
+    /// A Builder Class for the Working Day Service; used for easy configuration.
+    /// </summary>
+    public sealed class WorkingDayServiceBuilder
+    {
+        private readonly HashSet<NonWorkingDaySource> _sources;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="WorkingDayServiceBuilder"/> class.
+        /// </summary>
+        public WorkingDayServiceBuilder() => _sources = new HashSet<NonWorkingDaySource>();
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="WorkingDayServiceBuilder"/> class.
+        /// </summary>
+        /// <returns>The new instance of the <see cref="WorkingDayServiceBuilder"/> class.</returns>
+        public static WorkingDayServiceBuilder New() => new WorkingDayServiceBuilder();
+
+        /// <summary>
+        /// Builds a <see cref="WorkingDayService"/> with the configured set of <see cref="NonWorkingDaySource"/>s.
+        /// </summary>
+        /// <returns>A <see cref="WorkingDayService"/> which uses the <see cref="NonWorkingDaySource"/>s in the <see cref="WorkingDayServiceBuilder"/>'s <c>_sources</c>.</returns>
+        public WorkingDayService Build() => new WorkingDayService(_sources.ToList());
```

Comment:
You can make a builder that doesn't require you to call .Build() on the end with an implicit conversion operator ... something like:

```public static implicit operator WorkingDayService(WorkingDayServiceBuilder builder)
{
    return new WorkingDayService(builder.sources.ToList());
}```